nemo/collections/asr/models/aed_multitask_models.py

```diff
@@ -1044,6 +1096,8 @@ def _setup_transcribe_dataloader(self, config: Dict) -> 'torch.utils.data.DataLo
             # when using a list of audio files instead of a manifest (added from TranscrptionMixin)
             manifest_filepath = os.path.join(config['temp_dir'], 'manifest.json')
             batch_size = min(config['batch_size'], len(config['paths2audio_files']))
+        # check this part!
+        enable_parallel_chunking = batch_size == 1  # <-- enables chunking for batch size 1
```

Comment:
should have an option through `.transcribe(enable_chunking=False)` to disable chunking even when bs=1